files/cifs

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+set -u
+
+init() {
+	assertBinaryInstalled mount.cifs cifs-utils
+	assertBinaryInstalled jq jq
+	assertBinaryInstalled mountpoint util-linux
+	assertBinaryInstalled base64 coreutils
+	echo '{ "status": "Success", "message": "The fstab/cifs flexvolume plugin was initialized successfully", "capabilities": { "attach": false } }'
+	exit 0
+}
+
+assertBinaryInstalled() {
+	binary="$1"
+	package="$2"
+	if ! which "$binary" > /dev/null ; then
+		errorExit "Failed to initialize the fstab/cifs flexvolume plugin. $binary command not found. Please install the $package package."
```

Comment:
Package names are Ubuntu-specific; CoreOS doesn't have a pkg manager, so this error message isn't applicable.